MAUI/Cards/getting-started.md

```diff
@@ -0,0 +1,209 @@
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="1 5 6 7" %}
+
+<ContentPage   
+    . . .
+    xmlns:card="clr-namespace:Syncfusion.Maui.Cards;assembly=Syncfusion.Maui.Cards">
+
+<cards:SfCardView>
+    <Label Text="Label" Background="LightGoldenrodYellow" HorizontalTextAlignment="Center" VerticalTextAlignment="Center"/>
+</cards:SfCardView>
+
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="1 9 10 11" %}
+
+using Syncfusion.Maui.Cards;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfCardView cardView = new SfCardView();
+
+        cardView.Content = new Label()
+        {
+            Text = "Label",
+            HorizontalTextAlignment = TextAlignment.Center,
+            VerticalTextAlignment = TextAlignment.Center,
+            BackgroundColor = Colors.LightGoldenrodYellow
+        };
+        this.Content = cardView;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Register the handler
+
+The `Syncfusion.Maui.Core` NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.
+
+{% tabs %}
+{% highlight C# tabtitle="MauiProgram.cs" hl_lines="1 10" %}
+
+using Syncfusion.Maui.Core.Hosting;
+namespace GettingStarted
+{
+    public static class MauiProgram
+    {
+        public static MauiApp CreateMauiApp()
+        {
+            var builder = MauiApp.CreateBuilder();
+
+            builder.ConfigureSyncfusionCore();
+            builder
+            .UseMauiApp<App>()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+                fonts.AddFont("Segoe-mdl2.ttf", "SegoeMDL2");
+            });
+
+            return builder.Build();
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## SwipeToDismiss
+
+The `SwipeToDismiss` property enables or disables the swiping feature in the SfCardView.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="1" %}
+
+<cards:SfCardView SwipeToDismiss="true">
+    <Label Text="SfCardView" Background="LightGoldenrodYellow" HorizontalTextAlignment="Center" VerticalTextAlignment="Center"/>
+</cards:SfCardView>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="1 9 10 11" %}
+
+using Syncfusion.Maui.Cards;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfCardView cardView = new SfCardView();
+        cardView.SwipeToDismiss = true;
+        cardView.Content = new Label(){ Text="SfCardView" };
+        this.Content = cardView;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}	
+
+## Dismiss the card programmatically
+
+The `IsDismissed` property enables us to retrieve the view state of the card, and it also provides the capability for the user to programmatically modify the card's view state.
```

Comment:
The `IsDismissed` property enables you to retrieve the view state of the card, and also provides the capability for the user to programmatically modify the card's view state.